scripts/release

```diff
@@ -262,10 +267,15 @@ build_docker_image() {
   AGENT_IMAGE_NAME=$image_name make -C $SCRIPT_DIR/.. image
 }
 
-push_docker_image() {
-  local stage="$1"
-  local new_version="$2"
-  docker push $(docker_repo_from_stage $stage):$new_version
+update_floating_docker_tags() {
+  local new_version="$1"
+  local new_image="$2"
+  local image_base="$3"
+
+  major_version=$(echo -n "$new_version" | head -c1)
```

Comment:
Are we planning to change this script when we start having double digit major versions?

Reply:
To @seonsfx's point, maybe do `cut -d . -f 1` instead since it's future-proof?

Reply:
```suggestion
  major_version=$(echo -n "$new_version" | cut -d. -f1)
```